acceptance/auth/bundle_and_profile/script

```diff
@@ -2,6 +2,7 @@
 envsubst < databricks.yml > out.yml && mv out.yml databricks.yml
 envsubst < .databrickscfg > out && mv out .databrickscfg
 export DATABRICKS_CONFIG_FILE=.databrickscfg
+export DATABRICKS_AUTH_TYPE=pat
```

Comment:
commit message says "path" but the code says "pat". typo?

Also, could you add a comment why this must be set explicitly, what is selected otherwise and in which envs and what is the effect of that?

Reply:
Strictly speaking we don't need this and don't need to setup token in databrickscfg as well for this test, so removed both to keep it simpleStrictly speaking we don't need this and don't need to setup token in databrickscfg as well for this test, so removed both to keep it simple